src/common/context/GlobalContext.tsx

```diff
@@ -46,7 +46,7 @@ export const GlobalContext = createContext<GlobalContextProps>({
     networkId: ChainID.Mainnet,
     mode: NetworkModes.Mainnet,
   },
-  activeNetworkKey: NetworkModeUrlMap[NetworkModes.Mainnet],
+  activeNetworkKey: NetworkModeUrlMap[NetworkModes.Mainnet], // TODO: this is a confusing name as it's actually the url for the api based on the network...
```

Comment:
Just to provide some context, we chose this naming convention because we're using the API URL as the unique identifier (key) for the network. If you feel the current name is a bit confusing, please feel free to change it.

Reply:
cool. I will update in another PR